File: API/Entities/Order/Order.cs
using System;
using System.Collections.Generic;

namespace API.Entities.Order
{
    public class Order
    {
        public int Id { get; set; }
        public string UserId { get; set; }
        public ShippingAddress ShippingAddress { get; set; }
        public DateTime OrderDate { get; set; } = DateTime.Now;
        public List<OrderItem> OrderItems { get; set; }
        public long Total { get; set; }
        public OrderStatus OrderStatus { get; set; } = OrderStatus.Pending;
    }
}

File: shopping-cart/src/app/models/order.ts
export interface ShippingAddress {
        fullName: string;
        address1: string;
        area: string;
        city: string;
        state: string;
        zipcode: string;
        country: string;
    }

    export interface OrderItem {
        productId: number;
        name: string;
        imagePath: string;
        price: number;
        quantity: number;
    }

    export interface Order {
        id: number;
        userId: string;
        shippingAddress: ShippingAddress;
        orderDate: string;
        orderItems: OrderItem[];
        total: number;
        orderStatus: string;
    }